langchain/vectorstores/annoy.py

```diff
@@ -367,7 +367,7 @@ def from_embeddings(
                 from langchain import Annoy
                 from langchain.embeddings import OpenAIEmbeddings
                 embeddings = OpenAIEmbeddings()
-                db = Annoy.from_texts(texts, embeddings)
+                db = Annoy.from_embeddings(text_embeddings, embeddings)
```

Comment:
```suggestion
             text_embeddings = embedding.embed_documents(texts)
             db = Annoy.from_embeddings(list(zip(texts, text_embeddings)))
```

similar with below. The method expects a list of tuples rather than two lists, so it would look like [("hello world", [.1, .3.4]), ("foo", [.5, .2, .3]), ...]

Reply:
Added, thanks. The only difference to the suggestion is that I think the method still expects `Embeddings` object to be passed as well